release-1.8/release_notes_draft.md

```diff
@@ -123,6 +123,19 @@ pipeline.
 
 [SIG Instrumentation]: https://github.com/kubernetes/community/tree/master/sig-instrumentation
 
+### SIG Scalability
+
+[SIG Scalability][] is responsible for scalability testing, measuring/improving system
+performance and answering questions related to scalability.
+
+For the 1.8 release, SIG scalability focused on automation of large cluster
+scalability testing on CI + defining a concrete process around it, documentation
+of our current scalability thresholds and definition of a new set of SLIs/SLOs
+spanning across the system. Here's the release [scalability validation report].
+
+[SIG Scalability]: https://github.com/kubernetes/community/tree/master/sig-scalability
```

Comment:
Suggested rewrite for clarity:

SIG Scalability][] is responsible for scalability testing, measuring and improving system
performance, and answering questions related to scalability.
				
For the 1.8 release, SIG Scalability focused on automating large cluster
scalability testing on CI. In addition to defining a concrete process for scalability testing, SIG Scalability created documentation for the current scalability thresholds and defined a new set of SLIs and SLOs
spanning across the system. Here's the release [scalability validation report].